### PR TITLE
WF-62203-Add information about linking paragraph and character style in Word document

### PR DESCRIPTION
**Pull request by MathanKumarVaradhaRajaPerumal**
Task id : https://syncfusion.atlassian.net/browse/WF-62203 

Add information about linking paragraph and character style in Word document

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 1]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+
+{% tabs %} 
+
+{% highlight c# %}
+//Opens an input Word template
+WordDocument document = new WordDocument();
+//This method adds a section and a paragraph in the document
+document.EnsureMinimal();
+//Adds a new paragraph style named "MyParaStyle"
+IWParagraphStyle myParaStyle = document.AddParagraphStyle("MyParaStyle");
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2233]: Change the instance name as "paraStyle"
Change the Style name as "ParagraphStyle"

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 2]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+### Link Paragraph and Character Style
+
+You can link paragraph and character style using the `LinkedStyleName` property.
+
+The following code example explains how to link paragraph and character style.
+
+{% tabs %} 
+
+{% highlight c# %}
+//Opens an input Word template
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2228]: //Creates a Word document

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 3]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+WordDocument document = new WordDocument();
+//This method adds a section and a paragraph in the document
+document.EnsureMinimal();
+//Adds a new paragraph style named "MyParaStyle"
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2232]: Change the comments as per style name

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 4]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+(TryCast(document.LastParagraph.ChildEntities(0), WTextRange)).ApplyStyle("MyParaStyle")
+'Saves and closes the document instance
+document.Save(outputFileName, FormatType.Docx)
+document.Close()
+{% endhighlight %}
+
+{% highlight UWP %}
+//"App" is the class of Portable project.
+Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+using (WordDocument document = new WordDocument(assembly.GetManifestResourceStream("CreateWordSample.Assets.Test.docx"), FormatType.Docx))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2298]: Why opening Word document?

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 5]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+	//Please refer the below link to save Word document in UWP platform
+	//https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+	document.Close();
+}
+{% endhighlight %}
+
+{% highlight ASP.NET CORE %}
+FileStream sourceStreamPath = new FileStream(sourceFileName, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Opens an source document from file system through constructor of WordDocument class
+using (WordDocument document = new WordDocument(sourceStreamPath, FormatType.Automatic))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2337]: Change the code to create document

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 6]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
 
 {% endtabs %} 
 
+### Link Paragraph and Character Style
+
+You can link paragraph and character style using the `LinkedStyleName` property.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2221]: You can link character styles with  paragraph and vice versa in a Word document using `LinkedStyleName` property.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 7]
```diff
@@ -2216,6 +2216,186 @@ Please download the helper files from the following link to save the stream as a
...
+
+You can link character styles with paragraph and vice versa in a Word document using LinkedStyleName property.
+
+The following code example explains how to link character and paragraph style.
+
+{% tabs %} 
+
+{% highlight c# %}
+//Creates a Word document
+WordDocument document = new WordDocument();
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Word-document.md:2229]: Modify the code with using statement in all platforms

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 1 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 8]
```diff
@@ -2216,6 +2216,202 @@ Please download the helper files from the following link to save the stream as a
...
+//This method adds a section and a paragraph in the document
+document.EnsureMinimal();
+//Adds a new paragraph style named "MyParaStyle"
+IWParagraphStyle myParaStyle = document.AddParagraphStyle("MyParaStyle");
+//Sets the formatting of the style
+myParaStyle.CharacterFormat.FontSize = 16f;
+myParaStyle.CharacterFormat.TextColor = Color.DarkBlue;
+myParaStyle.ParagraphFormat.HorizontalAlignment = HorizontalAlignment.Center;
+//Adds a new character style named "MyCharStyle"
+IWCharacterStyle myCharStyle = document.AddCharacterStyle("MyCharStyle");
```
[the remark this answers is not in the document]
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 3 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 4 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed as creating

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 5 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 6 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: changed

File: File-Formats/DocIO/Working-with-Word-document.md [hunk 9]
```diff
@@ -2216,6 +2216,186 @@ Please download the helper files from the following link to save the stream as a
...
+//Appends the contents into the paragraph
+document.LastParagraph.AppendText("AdventureWorks Cycles");
+//Applies the style to paragraph
+document.LastParagraph.ApplyStyle("ParagraphStyle");
+//Appends new paragraph in section
+document.LastSection.AddParagraph();
+//Appends the contents into the paragraph
+document.LastParagraph.AppendText("AdventureWorks Cycles, the fictitious company on which the AdventureWorks sample databases are based, is a large, multinational manufacturing company.");
+//Applies style to the text range
+(document.LastParagraph.ChildEntities[0] as WTextRange).ApplyStyle("CharacterStyle");
```
[the remark this answers is not in the document]
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md:2252]: changed

File: File-Formats/DocIO/Working-with-Word-document.md (hunk 7 above)
Comment (MathanKumarVaradhaRajaPerumal) [reply File-Formats/DocIO/Working-with-Word-document.md]: modified with using statement

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**